### PR TITLE
Lab 1 for Review

### PR DESCRIPTION
**Pull request by gkovoor**
@rbattini1

### CODE REVIEW

**Comment by rbattini1** [comment]
Hey for 3B I was unable to figure out what the specific statistical question you were asking. Also 3C/3D I was unable to find the integral.

For 4 I think you should have more discussion in terms of testing new values and discussing the patterns.

Similarly for the non-continuous distributions try a couple different plots with different parameters with a hypothetical question as well as answering E.

**Comment by gkovoor** [comment]
@oliviatessa

File: LAB1.html [hunk 1]
```diff
@@ -0,0 +1,12362 @@
...
+.btn-info[disabled].focus,
+fieldset[disabled] .btn-info.focus {
+  background-color: #5bc0de;
+  border-color: #46b8da;
+}
+.btn-info .badge {
+  color: #5bc0de;
+  background-color: #fff;
+}
+.
```
Comment (oliviatessa) [inline LAB1.html]: So z score and sigma are the same thing? How are they related?

Comment (oliviatessa) [reply LAB1.html]: Where does the negative sign come from?

Comment (oliviatessa) [inline LAB1.html]: Standard deviation and 'sigma' as the instructor means it are not the same thing. Revisit the concept of sigma. It's important and will come up again in future labs

**Comment by oliviatessa** [comment]
Overall, your lab needs a lot more explanation. Think of labs as an essay. You are graded primarily on the quality of your explanations and interpretations of your results. Try to write as if to someone who does not have a background in statistics or know what your assignment is until they read your lab. 

For example, the first part of your lab as almost no explanation. Explain what a normal distribution is. Explain what an exponential distribution is. 

**Also, I think you need to revisit the concept of 'sigma' as a probability.**

**Pull request closed without merging**